aria-practices.html

```diff
@@ -2150,6 +2150,43 @@ <h4>WAI-ARIA Roles, States, and Properties</h4>
       </section>
     </section>
 
+    <section class="widget" id="meter">
+      <h3>Meter</h3>
+      <p>
+        A <a class="role-reference" href="#meter">meter</a> allows authors to define a value within a known range. The minimum and maximum limits of the range must be defined numerically, and the value should be a number within the range.
```

Comment:
I really like this definition, nice work!

Reply:
😊